packages/static-wado-deploy/bin/deploydicomweb.mjs

```diff
@@ -1,5 +1,6 @@
-import configureProgram from "../lib/program/index.mjs";
-import { deployConfig } from "../lib/index.mjs";
+#!/usr/bin/env node
```

Comment:
Is there a way to have this working as a executable, like mkdicomweb?
Also in a fresh clone executing this command raises error complaining about s3-deploy

Reply:
After you do all the npm install -g .
you need to run yarn link:exec

Reply:
Not worked... I noticed that when you save a key image you call this command that doesnt exist...

Reply:
Thank you for finding the capitalization issue that was causing the deploy issue on case sensitive file systems.